### PR TITLE
[XPipeline][HDRP]Refactor CameraUI - Rendering

### PR DESCRIPTION
**Pull request by alex-vazquez**
### Purpose of this PR
Clean Camera UI, and extract everything to it's own file and methods

Rendering section.
image

---
### Testing status
Test that everything is the same as before

---
### Comments to reviewers
Another step towards URP and HDRP camera editor unification

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Skin.cs [hunk 1]
```diff
@@ -51,23 +43,14 @@ class Styles
...
-
-            public static readonly GUIContent ditheringContent = EditorGUIUtility.TrTextContent("Dithering", "Should we apply 8-bit dithering to the final render?");
-            public static readonly GUIContent stopNaNsContent = EditorGUIUtility.TrTextContent("Stop NaNs", "Automatically replaces NaN/Inf in shaders by a black pixel to avoid breaking some effects. This will slightly affect performances and should only be used if you experience NaN issues that you can't fix.");
-
             public static readonly GUIContent allowDynResContent = EditorGUIUtility.TrTextContent("Allow Dynamic Resolution", "Whether to support dynamic resolution.");
 
             public static readonly GUIContent viewportContent = EditorGUIUtility.TrTextContent("Viewport Rect", "Four values that indicate where on the screen HDRP draws this Camera view. Measured in Viewport Coordinates (values in the range of [0, 1]).");
             public static readonly GUIContent depthContent = EditorGUIUtility.TrTextContent("Depth");
+
+#if ENABLE_VR && ENABLE_XR_MANAGEMENT
```
Comment (jenniferd-unity) [inline com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Skin.cs:51]: i am not sure the localization string should be excluded from the build - I hope it does not change how their system work. i think it does not?

**Review by jenniferd-unity — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Skin.cs (hunk 1 above)
Comment (RSlysz) [reply com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Skin.cs:51]: I think as it is not localized, it should be ok. But do a test and we will be sure of it.

**Review by RSlysz — APPROVED** [review]

**Comment by alex-vazquez** [comment]
@Unity-Technologies/gfx-qa-hdrp , could you test that nothing is broken on this section? Thank you :)

**Review by TomasKiniulis — APPROVED** [review]
Going through the list of all Rendering options and some Custom Frame Settings, all options still work as expected

**Comment by sebastienlagarde** [comment]
@alex-vazquez you need to re-merge amster.

**Pull request merged**